mapping/sustainability.yaml

```
@@ -783,8 +783,23 @@
     module: Climate finance
     indicator: Climate impact
     disclosure format: Disclose the cost per tonne of CO2 equivalent \[value, currency\].
-    mapping: Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in documents with `.documentType` set to 'abatementCostMethodology'.
-    example: ''
+    mapping: Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in `documents` with `.documentType` set to 'abatementCostMethodology'.
```

Copy edit and reference common operation.

```suggestion
    mapping: Map to `environment.abatementCost`. If supporting documentation is available, [add a project document](../common.md#add-a-project-document) and set `.documentType` to 'abatementCostMethodology'.
```